main.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "DB_HOST"
+	port     = 5432
+	user     = "GO_DB_USER"
+	password = "GO_DB_PASS"
+	name     = "GO_DB_NAME"
+	driver   = "postgres"
+)
+
+var (
+	db_host     = os.Getenv(host)
+	db_user     = os.Getenv(user)
+	db_password = os.Getenv(password)
+	db_name     = os.Getenv(name)
+	connStr     = fmt.Sprintf(
+		"host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
+		db_host, port, db_user, db_password, db_name,
+	)
+)
+
+type CityInfo struct {
+	CityName    string `db:"city_name"`
+	Temperature string `db:"temperature"`
+	Humidity    string `db:"humidity"`
+}
+
+var cI CityInfo
```

Comment:
![image](https://user-images.githubusercontent.com/669518/215056956-b92a2fff-800c-46cd-8661-f84f7cfeeea4.png)

Reply:
the problem is not in the name )

Reply:
but still name was ugly)